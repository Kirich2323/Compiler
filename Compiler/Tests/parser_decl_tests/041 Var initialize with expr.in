const
    m = 10;
    l = 12.0;
var
    a : integer = -(10 + 2)*3 + 100;
    b : float = 10.1 / 2.0;
    c : integer = m + m;
    d : float = -l;