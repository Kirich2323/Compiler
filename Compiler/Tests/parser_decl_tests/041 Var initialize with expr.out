main
----------
char                     type 
float                    type 
integer                  type 
m               const integer              10
l                 const float              12
a                         var         integer             64
b                         var           float           5.05
c                         var         integer             20
d                         var           float            -12